MozillaRustComponents/Sources/MozillaRustComponentsWrapper/Generated/Metrics/Metrics.swift

```diff
@@ -26,6 +26,122 @@ extension GleanMetrics {
         public static let info = BuildInfo(buildDate: DateComponents(calendar: Calendar.current, timeZone: TimeZone(abbreviation: "UTC"), year: 2025, month: 12, day: 9, hour: 5, minute: 18, second: 29))
     }
 
+    enum AdsClient {
```

Comment:
These changes should come from the AS. I suppose this will come in once we have merged: https://github.com/mozilla/application-services/pull/7111